app/services/suri_minter.rb

```diff
@@ -0,0 +1,56 @@
+require 'net/http'
+require 'uri'
+require 'errors'
+
+class SuriMinter < ActiveFedora::Noid::Minter::Base
+
+  protected
+
+  def next_id
+    retrieve_id
+  end
+
+  private
+
+  def valid?
```

Comment:
Did you add this method to make `#handle_response` more robust, or to help bring parity with other AF::Noid minters (that provide the `#valid?` method)?

Reply:
My thinking is more robust. In the event that (for some reason) we get a 201 result from SURI but the response body doesn't look like a DRUID.

Reply:
OK. This does cause a mismatch with other AF::Noid minters which have `#valid?` as a public method with an arity of 1 (vs. 0). If you're considering removing the AF::Noid machinery in a follow-on PR, though, I'm not inclined to care about this very much!

Reply:
Yup. Working through removing now.